.travis.yml

```diff
@@ -20,12 +20,14 @@ matrix:
   fast_finish: true
 
 before_script:
-  - composer update --prefer-source $PREFER_LOWEST
+  - if [ $RUN_PHPSTAN == "FALSE" ]; then composer update --prefer-source $PREFER_LOWEST; fi
+  - if [ $RUN_PHPSTAN == "TRUE" ]; then composer install --no-dev; fi
```

Comment:
Phpstan would be a dev dependency and wouldnt be instlaled?

Reply:
The next line installs phpstan.
I put this line here, because I think maybe phpstan will want to analyse places where the code calls methods in dependencies... and so in repos that actually have code, phpstan likely needs to have visibility of the dependencies.